cloudaux/aws/iam.py

```diff
@@ -163,6 +163,33 @@ def get_role_instance_profiles(role, client=None, **kwargs):
     ]
 
 
+@sts_conn('iam', service_type='client')
+@rate_limited()
+def get_role_managed_policy_documents(role, client=None, **kwargs):
+    """Retrieve the currently active policy version document for every managed policy that is attached to the role."""
+    policies = get_role_managed_policies(role, **kwargs)
+
+    policy_documents = zip(
+        (policy['name'] for policy in policies),
+        Parallel(n_jobs=20, backend="threading")(
```

Comment:
It took me a bit to wrap my head around this syntax.

Could the context manager be used in its place? https://pythonhosted.org/joblib/parallel.html#reusing-a-pool-of-workers

Reply:
I don't think using the Parallel context manager API buys us much in terms of readability here, but I will break up this inline statement to make it clearer.